Directory.Build.targets

```diff
@@ -95,7 +95,8 @@
     <PackageReference Update="Microsoft.Extensions.Logging.Debug" Version="$(ExtensionsVersion)" />
     <PackageReference Update="Microsoft.Extensions.Options" Version="$(ExtensionsVersion)" />
     <PackageReference Update="Microsoft.Extensions.Options.ConfigurationExtensions" Version="$(ExtensionsVersion)" />
-    <PackageReference Update="Microsoft.Extensions.WebEncoders" Version="$(ExtensionsVersion)" />
+
+    <PackageReference Update="Microsoft.Extensions.WebEncoders" Version="$(AspNetCoreVersion)" />
```

Comment:
Considering this is `Microsoft.Extensions` package, why are we changing the version to be `AspNetCoreVersion` variable?

Reply:
Well, the project is in the [aspnetcore](https://github.com/dotnet/aspnetcore) repo and versioning is aligned. 
I can leave a comment or move it to asp net section of this file.

Reply:
Ahh, got it. No need to move, thx.